README.md

```diff
@@ -804,22 +804,26 @@ Scenario has secrets. Here's how you use them.
 state = scenario.State(
     secrets={
         scenario.Secret(
-            {0: {'key': 'public'}},
-            id='foo',
-        ),
-    },
+            current={'key': 'public'},
+            latest={'key': 'public', 'cert': 'private'},
```

Comment:
I'm a little confused, how can "current" and "latest" be different?

Reply:
current is 'what you currently are tracking'